docs/api.swagger.yaml

```diff
@@ -169,7 +233,6 @@ paths:
           description: The id of the OAuth 2.0 Client.
           name: id
           in: path
-          required: true
```

Comment:
@arekkas I'm not sure what was going on with the swagger changes here, I just ran the `gen-swagger.sh` and ran it through the conversion. I was hoping you knew lol